src/fides/api/util/tcf/experience_meta.py

```diff
@@ -1,32 +1,49 @@
 import hashlib
 import json
-from typing import Dict, List
+from typing import Dict, List, Union
 
-from pydantic import Extra, root_validator
+from pydantic import Extra, Field, root_validator
 
 from fides.api.models.privacy_notice import UserConsentPreference
 from fides.api.schemas.base_class import FidesSchema
 from fides.api.schemas.privacy_experience import ExperienceMeta
-from fides.api.schemas.tcf import TCMobileData
+from fides.api.schemas.tcf import (
+    TCFVendorConsentRecord,
+    TCFVendorLegitimateInterestsRecord,
+    TCMobileData,
+)
 from fides.api.util.tcf.tc_mobile_data import build_tc_data_for_mobile
 from fides.api.util.tcf.tc_model import TCModel, convert_tcf_contents_to_tc_model
 from fides.api.util.tcf.tcf_experience_contents import TCFExperienceContents
 
 
 class TCFVersionHash(FidesSchema):
-    """Minimal subset of the TCF experience details that capture when consent should be resurfaced"""
+    """Minimal subset of the TCF experience details that capture when consent should be resurfaced
 
-    policy_version: int
-    purpose_consents: List[int]
-    purpose_legitimate_interests: List[int]
-    special_feature_optins: List[int]
-    vendor_consents: List[int]
-    vendor_legitimate_interests: List[int]
+    These values are later hashed, and you can treat a change in a hash as a trigger for re-establishing
+    transparency and consent
+    """
+
+    policy_version: int = Field(
+        description="TCF Policy Version. A new policy version invalidates existing strings and requires CMPs to "
+        "re-establish transparency and consent from users."
+    )
+    vendor_purpose_consents: List[str] = Field(
+        description="Stores vendor * purpose * consent legal basis in the format "
+        "<universal_vendor_id>-<comma separated purposes>"
+    )
+    vendor_purpose_legitimate_interests: List[str] = Field(
+        description="Stores vendor * purpose * legitimate interest legal basis in the format"
+        " <universal_vendor_id>-<comma separated purposes>"
+    )
```

Comment:
nit - any reason the spacing isn't consistent between the two here? just to make it all a bit less ambiguous, maybe do something like:
```suggestion
        description="Stores vendor * purpose * consent legal basis in the format: "
        "<universal_vendor_id>-<comma separated purposes>"
    )
    vendor_purpose_legitimate_interests: List[str] = Field(
        description="Stores vendor * purpose * legitimate interest legal basis in the format: "
        "<universal_vendor_id>-<comma separated purposes>"
    )
```

i wouldn't normally nitpick this much, but we don't have any "specs" for the values here outside of the code, so i figure it's worth removing any room for ambiguity!

Reply:
Thanks for the cleanup, just accidentally broke the lines in two separate places